drivers/rf-transceiver/ad9361/ad9361.c

```diff
@@ -5695,11 +5695,11 @@ int32_t ad9361_do_calib_run(struct ad9361_rf_phy *phy, uint32_t cal,
 		break;
 	}
 
-	ret = ad9361_tracking_control(phy, phy->bbdc_track_en,
-				      phy->rfdc_track_en, phy->quad_track_en);
+	ret2 = ad9361_tracking_control(phy, phy->bbdc_track_en,
```

Comment:
why not just check `ret` after the switch..case scenario?

Reply:
If the cal fails then we need to restore state.

Reply:
hmm, conditional jump can works also. but i guess it is fine like this too.